extensions/cornerstone/src/components/WindowLevelActionMenu/WindowLevelActionMenu.tsx

```diff
@@ -15,8 +15,9 @@ import { VolumeRenderingOptions } from './VolumeRenderingOptions';
 import { ViewportPreset } from '../../types/ViewportPresets';
 import { VolumeViewport3D } from '@cornerstonejs/core';
 import { utilities } from '@cornerstonejs/core';
+import { useSystem } from '@ohif/core';
 
-export const nonWLModalities = ['SR', 'SEG', 'SM', 'RTSTRUCT', 'RTPLAN', 'RTDOSE'];
+export const nonWLModalities = ['SR', 'SEG', 'SM', 'RTSTRUCT', 'RTPLAN'];
```

Comment:
Instead of having a list like this, can you ask the display set if it is window levellable?

Reply:
Just that it really isn't a property of the window level action but a property of a display set.